examples/terraform-gcp-example/main.tf

```diff
@@ -0,0 +1,47 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# CONFIGURE THE GOOGLE CLOUD PROVIDER
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "google" {
+  project = "${var.project}"
+  version = "~> 1.16"
```

Comment:
Yep I agree, let me change.

Reply:
Turns out this will only work as long as the user has one of the Environment variables set: https://www.terraform.io/docs/providers/google/index.html. I might need to clarify somewhere in the docs.

Reply:
I was using `GOOGLE_CLOUD_PROJECT_ID`. I've now changed the GCP code to prefer: `GOOGLE_CLOUD_PROJECT` as it plays better with other tools like Terraform.